### ISSUE TITLE
NaN values in the output coordinates

### ISSUE THREAD

**Issue by f-meireles** [sokrypton/ColabFold#243]
## Expected Behavior
Greetings everyone,
I am trying to use a custom template to guide the building of a homooligomeric model.
## Current Behavior
But for some reason the models that are "built" are lacking all of the coordinates, presenting only NaN values. This happened to me both in my local colabfold installation and in the colabfold notebook: 
![image](https://user-images.githubusercontent.com/107609480/174012965-7bc33c1c-966a-43c0-b496-62137cfe0a80.png)

This problem only happens when I try to run homooligomers. If I try to predict a single chain of the same input sequence the software works properly:
![image](https://user-images.githubusercontent.com/107609480/174014968-cfe04ef5-2dc4-4783-85c5-1df144828e62.png)

I tested it with several fragments of the sequence that I am using and several numbers of chains (3,5,6,8).

Thank you in advance for your attention and please let me know if I can provide more useful info.

**Comment by f-meireles** [comment]
Hi @martin-steinegger , no problem! My input sequence is: PRDGYKFSLSDTVNKSDLNEDGTININGKGNYSAVMGDELIVKVRNLNTNNVQEYVIPVDKK

I am currently modeling the sequence through the older version of the notebook. One important thing that I forgot to mention: This problem that I had only happens when I try using **custom templates**. If I try to predict the homooligomers without any custom templates it does work properly.

**Comment by f-meireles** [comment]
I actually had the same problem in the older version:
![image](https://user-images.githubusercontent.com/107609480/174026004-5260df5e-40a4-4f97-bcf4-2e849f4325f3.png)

But interestingly the output error seems to be different this time.

**Comment by martin-steinegger** [comment]
That's good news in some way. Thank you so much. 
I assume your template might cause the issue. Could you share your input template?

**Comment by martin-steinegger** [comment]
Thank you so much. I tried to run your sequence with your custom template but it did not crash. 
Also using just templates ran without any issue. Did you use `6rb9.cif` or `6rb9.cif.txt`?

**Comment by f-meireles** [comment]
Oh, just to clarify, I can actually predict the monomer using templates with the sequence I posted above, what I can't do is predict the oligomers:
PRDGYKFSLSDTVNKSDLNEDGTININGKGNYSAVMGDELIVKVRNLNTNNVQEYVIPVDKK:PRDGYKFSLSDTVNKSDLNEDGTININGKGNYSAVMGDELIVKVRNLNTNNVQEYVIPVDKK 
Like this one.

**Comment by martin-steinegger** [comment]
Oh this indeed causes the same issue. It is quite odd. if I run it with a cpu I do get a result but a pretty bad one. 
```
2022-06-16 23:12:52,365 model_3 took 271.1s (3 recycles) with pLDDT 38.5, ptmscore 0.23 and iptm 0.0855
```
Somehow the template information is harmful. The aligned coordinates seem to be quite fragmented too.

**Comment by dxzh0129** [comment]
I have the same issue as f-meireles reported (NaN coordinates with homooligomer mode) with completely different input sequences. I used the batch notebook.

**Comment by charwich** [comment]
@martin-steinegger @ykagaya noted the JAX version in `pyproject.toml` made a big leap to 0.3.8, is this correct? The latest from DeepMind is on 0.2.14. Not sure we expect this to introduce NaNs.

**Comment by martin-steinegger** [comment]
Yes, it seem to be a jax related issue. We will downgrade it again soon.

**Comment by sokrypton** [comment]
jax 0.3.7 works
jax 0.3.8 does NOT work...

something broke in jax 0.3.8...

**Comment by sokrypton** [comment]
@YoshitakaMo found the source of the bug. We submitted a bug report!
https://github.com/deepmind/alphafold/issues/513
Hopefully, this will get fixed quickly, as downgrading jax to old version will slow down the google-colab notebook...

**Comment by martin-steinegger** [comment]
We fixed it in colabfold. Please give it a try.

**Comment by martin-steinegger** [comment]
Did you try the notebook? Yeah maybe wiping everything locally might help.

**Comment by f-meireles** [comment]
I just tried it and it and the notebook does work. I will try to see what is wrong in my local installation
